.github/workflows/push-container.yaml

```diff
@@ -16,20 +16,22 @@ on:
 jobs:
     push-container:
         runs-on: ubuntu-latest
+        env:
+            CAN_PUSH: ${{ (github.event_name == 'push' && github.repository == 'nangohq/nango') || (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false) }}
         steps:
             - uses: actions/checkout@v4
             - uses: docker/setup-buildx-action@v3
             - uses: actions/setup-node@v3
               with:
                 node-version: '18.x'
             - uses: docker/login-action@v3
-              if: github.repository == 'nangohq/nango'
+              if: env.CAN_PUSH
               with:
                 username: ${{ secrets.DOCKER_USERNAME }}
                 password: ${{ secrets.DOCKER_PASSWORD }}
             - name: Build and push container
               env:
-                PUSH: ${{ (github.repository == 'nangohq/nango') && '--push' || '' }}
+                PUSH: ${{ (env.CAN_PUSH) && '--push' || '' }}
```

Comment:
what is this `--push` || '' logic?

Reply:
--push is shorthand for `--output=type=docker`, basically telling docker buildx to push the image to the registry once built. Without an specified output the image is just built

More info at https://docs.docker.com/engine/reference/commandline/buildx_build/#docker